formats/application/vnd/oada/isoblue/1/+json/examples/default.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  _type: 'application/vnd.oada.isoblue.1+json',
+
+  context: {},
+
+  // The CAN key holds CAN messages logged by the ISOBlue can logging module
+  'can': {
+    _id: 'kfj20ikejldss',
+    _rev: '9-kfjo2i3jfelkfas',
```

Comment:
I missed this the first time: our _rev's used to look like this, now they are just integers.  Fix the _rev's in all your example links

Reply:
actually, your schema has these as just links, not versioned links.  I think they should not be versioned, so the schema is correct but the example is wrong because it still has the _rev.

Reply:
Fixed in 5696ced